library_generation/test/resources/goldens/pr_description-golden.txt

```diff
@@ -0,0 +1,18 @@
+BEGIN_COMMIT_OVERRIDE
+BEGIN_NESTED_COMMIT
+Update repo-level parameter gapic_generator_version to 1.2.3
+END_NESTED_COMMIT
+BEGIN_NESTED_COMMIT
+Update repo-level parameter libraries_bom_version to 2.3.4
+END_NESTED_COMMIT
+BEGIN_NESTED_COMMIT
+Update repo-level parameter protoc_version to 3.4.5
```

Comment:
We should remove protoc_version from our test proto now.

Reply:
Yes, but using protoc_version might also confuse other people. Can you use googleapis_commitish instead? Or use something random to indicate that the current logic can be used for any repo level changes?

Reply:
> Can you use googleapis_commitish instead?

googleapis commit will not appear here because the commits are formatted to separate messages.

I'll remove this message.

Reply:
Done